packages/inject/docs/readme-review-2026-01-28.md

```
@@ -94,13 +94,13 @@ Add Error Types section:
 
 Current:
 ```bash
-npm install @analog-tools/inject
+pnpm add @analog-tools/inject
 ```
 
 Recommended:
 ```bash
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)